docs/self-managed/setup/upgrade.md

```diff
@@ -9,7 +9,7 @@ import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
 :::note
-When upgrading to a new version of the Camunda 8 Helm charts, we recommend updating to the **latest patch** release of the **next major version**.
+When upgrading to a new version of the Camunda 8 Helm charts, we recommend updating to the **latest patch** release of the next **major** version of the chart.
```

Comment:
@conceptualshark What do you think about giving an example here?
So, adding something like:

> For example, If the current Helm chart version is 10.x.x and the latest next major version is 11.0.1, then it's recommended to upgrade to 11.0.1, not 11.0.0.

But that's just a suggestion and it's up to you.